src/SOS/Strike/disasmARM.cpp

```diff
@@ -355,7 +355,8 @@ void ARMMachine::Unassembly (
     GCEncodingInfo *pGCEncodingInfo, 
     SOSEHInfo *pEHInfo,
     BOOL bSuppressLines,
-    BOOL bDisplayOffsets) const
+    BOOL bDisplayOffsets,
+    std::function<void(ULONG*, UINT*, BYTE*)> displayIL) const
```

Comment:
We should probably use `displayIL` in the body?

Reply:
Yes. That was missed earlier. Thanks but I am not sure if that is linked to the build system at present.